src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Association Class 
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.